### DISCUSSION TITLE
Sequential Bayes Factor with bayestestR

### DISCUSSION THREAD

**Discussion by lucaonnis** [easystats/bayestestR#599] [Theoretical contemplations]
Is it possible/appropriate to apply the bayesfactor_models() function to to sequential designs, where the number of observations depends on an interim assessment of the evidence collected so far (Schönbrodt & Wagenmakers, 2018; Schönbrodt et al., 2017)? This would be useful for starting a human experiment with a given sample size N, and choosing a stopping rule based on the output of bayesfactor_models(), e.g.:

bayesfactor_models(m1, m2, denominator = m1)

The paper by Schönbrodt et al papers and other ones I perused indicate a method for sample size estimation and stopping rule criteria that apply to *t-test designs*, while I am testing linear mixed-effects models (m1 and m2 above).

**Comment by mattansb** [comment]
Hi Luca,

The idea of sequential analysis is not unique to Bayesian t-test Bayes-factors, and can be applied to essentially any testing framework - even significance testing (which does require some adjustments, [e.g.](https://doi.org/10.1198/016214508000000986)).

Therefore, computing Bayes factor sequentially as data is collected is a valid analysis choice.
